docs/best_practices.rst

```diff
@@ -0,0 +1,140 @@
+  generic and can be heavy-handed.
+* Client side JavaScript frameworks should be used as an enhancement, not as the 
+  default backbone for delivering content. Large frameworks such as React or Angular 
+  should  be used only when they are jointly identified as being the appropriate 
+  solution for  a specific requirement.
+* Images must be responsive through any combination of CSS, srcset, or SVG. Avoid 
+  particularly large images or a large number of images on a page (even if each one 
+  is relatively small). Production-ready images should also be optimized and/or 
+  compressed as appropriate.
+
+A good way to avoid a poorly performing web page is to set a performance budget. 
+An optimal goal is to deliver pages at or under a total of 600KB. A reasonable 
+breakdown for potential assets would be:
+
+* Fonts - 200KB
+* Images - 200KB
+* JS - 100KB
+* CSS - 20KB
+
+Before you enter the prototyping phase of your project, agree a performance budget 
+for your pages and then monitor throughout the development stage.
+
+
+Accessibility
+-------------
+
+Strive to make all Mozilla websites reasonably accessible to people with
+disabilities and those using assistive technologies. In many cases this isn't
+a matter of adding accessibility features into a site, but simply *not* adding
+obstacles that make the site harder to use. Accessible sites are better
+for everyone, not only those with disabilities.
+
+A few guidelines:
+
+* Include descriptive ``alt`` text for any meaningful images in HTML.
+* Include ``<label>`` elements in forms. Don't use ``placeholder`` as a label.
+* Navigation, forms, and interactive widgets like dropdown menus and modal windows
+  should be keyboard accessible.
+* Ensure sufficient color contrast so text is readable for most users, including
+  those with color vision deficiencies.
+* Text should be resizable for those who need larger type. Make reasonable allowances
+  for text to wrap and reflow when its size changes.
+* Include ARIA_ attributes in HTML where appropriate.
+
+.. _ARIA: http://www.w3.org/TR/wai-aria/
+
+
+Localization
+------------
+
+Many Mozilla websites are localized in a large number of locales and different languages. 
+If your project has localization (l10n) considerations then there are a few guidelines to 
+follow:
+
+* One of the biggest design challenges is ensuring text is both live and expandable. 
+  Text included in images or videos is prohibitively time consuming (if not impossible) to 
+  localize. The amount of actual text in a given message can vary greatly across languages, 
+  meaning the height and width of the surrounding elements must be able to adapt. Precise 
+  typography – such as controlled alignment, fitting text in an unusual shape, or specifying 
+  where lines should break when text wraps – is often unrealistic. Expect the shape and size 
+  of text blocks to change.
+* Choose web fonts with broad character sets. Many display fonts only support Roman alphabets 
```

Comment:
Do we have specific fonts we recommend with broad support for character sets? If so let's suggest them.